test/audiovideocontroller/DefaultAudioVideoController.test.ts

```diff
@@ -756,6 +757,52 @@ describe('DefaultAudioVideoController', () => {
       await stop();
     });
 
+    it('can be started and take a bandwidth update without update transceiver controller method', async () => {
+      class TestVideoUplinkBandwidth extends NoVideoUplinkBandwidthPolicy {
+        hasBandwidthPriority: boolean = false;
+
+        setHasBandwidthPriority(hasBandwidthPriority: boolean): void {
+          this.hasBandwidthPriority = hasBandwidthPriority;
+        }
+
+        maxBandwidthKbps(): number {
+          if (this.hasBandwidthPriority) {
+            return 100;
+          }
+          return 0;
+        }
+      }
+      const policy = new TestVideoUplinkBandwidth();
+      configuration.videoUplinkBandwidthPolicy = policy;
+      audioVideoController = new DefaultAudioVideoController(
+        configuration,
+        new NoOpDebugLogger(),
+        webSocketAdapter,
+        new NoOpMediaStreamBroker(),
+        reconnectController
+      );
+      await start();
+      await delay(defaultDelay);
+      audioVideoController.handleHasBandwidthPriority(true);
+    });
+
+    it('can be started and take a bandwidth update with update transceiver controller method', async () => {
```

Comment:
Where is your test that if you create a meeting and call start, then stop, then start, then stop, `setTransceiverController` is called four times with appropriate parameters?